lib/releaselib.sh

```diff
@@ -460,8 +469,6 @@ release::gcs::copy_release_artifacts() {
   logrun rm -rf $gcs_stage || return 1
   logrun mkdir -p $gcs_stage || return 1
 
-  logecho "Publish release artifacts to gs://$bucket..."
-
   # Stage everything in release directory
   logecho "- Staging locally to ${gcs_stage##$build_output/}..."
```

Comment:
PR builds currently rely on this side effect (the local `gcs-stage` directory) for the kubernetes release tarballs.

With this change, this local staging dir won't be created, so re-runs of a PR (e.g. when SQ runs again before merge) won't work.